llmproxy/tests/unit_tests/models/test_openai.py

```diff
@@ -0,0 +1,43 @@
+import os
+
+from dotenv import load_dotenv
+from llmproxy.models.openai import OpenAI
+from openai import error
+from unittest.mock import patch
+
+load_dotenv(".env.test")
+
+openai_api_key = os.getenv("OPENAI_API_KEY")
+
+def test_invalid_temperature():
+    chatbot = OpenAI(api_key = openai_api_key, temp=-1)
+    response = chatbot.get_completion()
+    assert isinstance(response.err, str)
+    assert "temperature" in response.message.lower()
```

Comment:
When you pass in an key with expired tokens, that is the error that will be returned, so no temperature error exists.
Feel free to add a temperature check, it is a valid unit test

Reply:
After some research, we realized you can't pass in an invalid temperature because any integer will be clamped to the range 0.0 - 1.0

Ex 1: 100 will become 1.0
Ex 2: -100 will become 0.0